Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -2603,6 +2609,32 @@ - (void)tappedAddToContactsOfferMessage:(OWSAddToContactsOfferMessage *)errorMes
                                                         editImmediately:YES];
 }
 
+- (void)tappedAddToProfileWhitelistOfferMessage:(OWSAddToProfileWhitelistOfferMessage *)message
+{
+    UIAlertController *alertController =
+        [UIAlertController alertControllerWithTitle:nil message:nil preferredStyle:UIAlertControllerStyleActionSheet];
+
+    UIAlertAction *leaveAction = [UIAlertAction
```

Comment:
copy/paste error? "leaveAction" -> "approveAction"

Also I think this should be UIAlertActionStyleDefault